src/components/DashboardSidebar.jsx

```diff
@@ -1,21 +1,20 @@
-import React, { useContext } from 'react';
-import { useSelector } from 'react-redux';
+import {useContext} from 'react';
+import {useSelector} from 'react-redux';
 
 import PropertyGrid from './PropertyGrid';
 import MapPanel from './MapPanel';
-import LanguageSwitch from './LanguageTest';
 
 import FinishTile from './FinishTile';
-import { MapContext } from './contexts/MapContextProvider';
-import { selectLoggedInUserRoles } from '../redux/selectors/authSelector';
-import { roles } from '../utilities/constants';
+import {MapContext} from './contexts/MapContextProvider';
+import {selectLoggedInUserRoles} from '../redux/selectors/authSelector';
+import {roles} from '../utilities/constants';
 
 import './../stylesheets/dashboardSidebar.scss';
 
-const { REACT_APP_MVP_DISABLED } = process.env;
+// const { REACT_APP_MVP_DISABLED } = process.env;
```

Comment:
If we do not need it anymore - delete it

Reply:
Ok